meshapp/src/home/login.tsx

```diff
@@ -52,6 +53,18 @@ const SignUp = () => {
 };
 
 const Login = () => {
+
+  const [showForgotPassword, setShowForgotPassword] = useState(false);
+
+  const updateShowForgotPasswordState = () => {
+    if (showForgotPassword == false) {
+      setShowForgotPassword(true)
+    }
+    if (showForgotPassword == true) {
+      setShowForgotPassword(false)
+    }
+    
```

Comment:
1. This should be done without if-statements. 
2. If you really want to keep the if-statements, this should be if/else. `showForgotPassword` cannot be both true and false. An interviewer probably wouldn't be happy if they saw this. 

https://softwareengineering.stackexchange.com/questions/12807/make-a-big-deal-out-of-true

Reply:
Just addressed this, thank you for that!